### PR TITLE
YLDs due to other causes background and motivation

### PR DESCRIPTION
**Pull request by alibow**
In looking into the GBD COMO adjustment methods I realized there are some issues with the proposal to address this issue on this document: https://vivarium-research.readthedocs.io/en/latest/models/causes/other_causes/index.html. I am planning to explore possible solutions to dealing with the issue, but not in this PR :). This PR outlines the GBD COMO methodology and how we are inconsistent with this methodology by default in vivarium.

### CODE REVIEW

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+GBD methodology
+++++++++++++++++
+
+Disability weight estimation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The GBD study relies on disability weights (DWs) to estimate years lived with 
+disability (YLDs). DWs represent the magnitude of health loss associated with a 
+specifc health outcome and are measured on a scale from 0 to 1; 0 implies a
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:54]:
```suggestion
specific health outcome and are measured on a scale from 0 to 1; 0 implies a 
```

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+Disability weight estimation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The GBD study relies on disability weights (DWs) to estimate years lived with 
+disability (YLDs). DWs represent the magnitude of health loss associated with a 
+specifc health outcome and are measured on a scale from 0 to 1; 0 implies a 
+state equivalent to full health and 1 equivalent to death.
+
+Disability weights have been estimated through a series of surveys, described 
+in deteail in the GBD methods appendix along with a complete listing of
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:58]:
```suggestion
in detail in the GBD methods appendix along with a complete listing of 
```

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+state equivalent to full health and 1 equivalent to death.
+
+Disability weights have been estimated through a series of surveys, described 
+in deteail in the GBD methods appendix along with a complete listing of 
+disability weight values for all of the health states included in the GBD study.
+
+Comorbidity correction
+~~~~~~~~~~~~~~~~~~~~~~
+
+The disability weights estimated through the process described above considers
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:64]:
```suggestion
The disability weights estimated through the process described above consider
```

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+sequela prevalence.
+
+.. note::
+
+  The COMO process in GBD assumes total independence between health states and 
+  does not account for any correlation that may exist between related health 
+  states (beyond correlation induced by location, age, and sex, which is 
+  accounted for through its stratified adjustments).
+
+  As reported in the GBD 2019 methods appendix, "[they] tested the contribution
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:83]:
```suggestion
  As reported in the GBD 2019 methods appendix, "[GBD] tested the contribution 
```

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+For each simulant, the overall disability weight is calculated as the 
+multiplicative combination of each of the sequela that the simulant possesses 
+according to the following equation:
+
+.. math::
+
+  DW_\text{overall} = 1 - \prod_{k=i}^j (1 - DW_k)
+
+Where: :math:`DW_k` is the *Kth* disease sequelae that the simulant possesses.
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:100]:
```suggestion
Where: :math:`DW_k` is the *:math:`k`-th* disease sequelae that the simulant possesses.
```

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+Once this overall disability weight is computed for each individual simulant, 
+it is the disaggregated into its component parts once more to obtain
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:103]:
```suggestion
it is then disaggregated into its component parts once more to obtain 
```

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+it is the disaggregated into its component parts once more to obtain 
+cause-specific disability weights adjusted for comorbidity according to the 
+following formula:
+
+.. math::
+
+  ADW_k = \frac{DW_k}{\sum_{k=i}^j DW_k} \times DW_\text{overall}
+
+Where: :math:`ADW_k` is the comorbidity-adjusted disability weight for sequela 
+*K* among a given simulant.
```
Comment (aflaxman) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:112]:
```suggestion
*:math:`k`* among a given simulant.
```

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 8]
```diff
@@ -0,0 +1,230 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The GBD study relies on disability weights (DWs) to estimate years lived with 
+disability (YLDs). DWs represent the magnitude of health loss associated with a 
+specific health outcome and are measured on a scale from 0 to 1; 0 implies a 
+state equivalent to full health and 1 equivalent to death.
+
+Disability weights have been estimated through a series of surveys, described 
+in detail in the GBD methods appendix along with a complete listing of 
+disability weight values for all of the health states included in the GBD study.
```
Comment (pletale) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:59]:
```suggestion
disability weight values for all health states included in the GBD study.
```

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+
+Comorbidity correction
+~~~~~~~~~~~~~~~~~~~~~~
+
+The disability weights estimated through the process described above consider
+each health state individually and must be corrected for the simultaneous 
+presence of multiple disability-causing health states within a given 
+individual. The GBD study performs this comorbidity correct (termed "COMO")
```
Comment (pletale) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:67]:
```suggestion
individual. The GBD study performs this comorbidity correction (termed "COMO") 
```
This probably makes no difference but this version feels less jargony to me somehow?

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 10]
```diff
@@ -0,0 +1,230 @@
...
+
+Then, cause-specific YLDs per capita in an age-sex-location-year are computed 
+as the sum of the adjusted DWs for each sequela across all *n* simulants in the 
+microsimulation as:
+
+.. math::
+  
+  \text{YLD rate}_k = \frac{\sum_{simulant=1}^n ADW_k}{n}
+
+The implication of the COMO correction for GBD YLDs attributable to a parent
```
Comment (pletale) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:123]:
```suggestion
The implication of the COMO adjustment for GBD YLDs attributable to a parent 
```
assuming saying como correction is redundant

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 11]
```diff
@@ -0,0 +1,230 @@
...
+
+  - Cannot calculate cause-specific YLDs adjusted for comorbidity even just among modeled causes, resulting in overestimation of cause-specific YLDs relative to "all cause" YLDs among modeled causes
+
+  - Does not adjust for comorbidity due to unmodelled causes, resulting in overestimation of YLDs due to modeled causes relative to GBD estimates
+
+- Underestimation of total YLDs
+
+  - Only observe a subset of total YLDs within our simulation. While we can calculate YLDs averted between scenarios, we cannot accurately calculate percent reduction in all-cause YLDs or DALYs relative to baseline because we do not model YLDs due to all causes at baseline.
+
+  - Causes us to overestimate impact of a death averted in our simulation. Death averted some number of YLLs averted, but really this person should then start accruing YLDs overtime, which will decrease the number of DALYs averted relative to baseline.
```
Comment (pletale) [inline docs/source/model_design/vivarium_features/other_causes_ylds/index.rst:194]:
```suggestion
  - Causes us to overestimate impact of a death averted in our simulation. Death averted allow some number of YLLs averted, but really this person should then start accruing YLDs overtime, which will decrease the number of DALYs averted relative to baseline. 
```
Not sure if this is the best word choice but a verb seems to be missing here? Unless I am misreading this sentence.

**Review by pletale — APPROVED** [review]
Looks good! Just added a few grammatical suggestions, but feel free to take them or leave them (-:

**Review by SylLutze — APPROVED** [review]

File: docs/source/model_design/vivarium_features/other_causes_ylds/index.rst [hunk 12]
```diff
@@ -0,0 +1,230 @@
...
+
+Comorbidity correction
+~~~~~~~~~~~~~~~~~~~~~~
+
+The disability weights estimated through the process described above consider
+each health state individually and must be corrected for the simultaneous 
+presence of multiple disability-causing health states within a given 
+individual. The GBD study performs this comorbidity correct (termed "COMO") 
+using a microsimulation. 
+
```
[the remark this answers is not in the document]
Comment (alibow) [reply docs/source/model_design/vivarium_features/other_causes_ylds/index.rst]: Good suggestion! I added a sentence to this effect

**Pull request merged**